SliderButton/SliderButtonBehaviorDecorator.js

```diff
@@ -0,0 +1,48 @@
+import platform from '@enact/core/platform';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {findDOMNode} from 'react-dom';
+
+// Adds agate-specific SliderButton behaviors
+// * aria-valuetext handling
+const SliderButtonBehaviorDecorator = (Wrapped) => {
+	const useValueText = ({onChange, ...rest}) => {
+		const {children} = rest;
+		const [valueText, setValueText] = React.useState(children[0]);
```

Comment:
Do we need to check if children is not undefined?
If getting the children texts from a server, it could be `undefined` at the first time.

Reply:
good point